package.json

```diff
@@ -6,8 +6,8 @@
   "types": "dist/index.d.ts",
   "license": "MIT",
   "scripts": {
-    "build": "bunchee",
-    "test": "NODE_ENV=test dotenv -e .env jest"
+    "build": "npx bunchee",
```

Comment:
No need `npx`, bunchee is in the devDeps.

```suggestion
    "build": "bunchee",
```

Reply:
That is well then. I needed npx on my machine but that's okay.